restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -281,6 +283,27 @@ public CallManager(final Configuration configuration, final ServletContext conte
         if (logger.isInfoEnabled()) {
             logger.info("CallManager extensions: "+(extensions != null ? extensions.size() : "0"));
         }
+        firstTimeCleanup();
+    }
+
+    private void firstTimeCleanup() {
+        if (logger.isInfoEnabled())
+            logger.info("Initial CallManager cleanup. Will check running state calls in DB and update state of the calls.");
+        String instanceId = RestcommConfiguration.getInstance().getMain().getInstanceId();
+        Sid sid = new Sid(instanceId);
+        final CallDetailRecordsDao callDetailRecordsDao = storage.getCallDetailRecordsDao();
+        List<CallDetailRecord> results= callDetailRecordsDao.getInCompleteCallDetailRecordsByInstanceId(sid);
+        for (CallDetailRecord result : results) {
```

Comment:
We are having UserAgentmanager class, would be the same as this one and it use intergradtion test for it. my new function  firstTimeCleanup is void function. I still dont know how to create unit test for it. It would be nice if you can give my some idea on how to create unit test here.